app/components/attachment/delete_form_component.rb

```diff
@@ -0,0 +1,7 @@
+# Display a form for destroying a file attachment via a button, but since it might already be nested within a form
+# put this component before the actual form containing the editcomponent
+class Attachment::DeleteFormComponent < ApplicationComponent
+  def call
+    form_tag('/attachments/:id', method: :delete, data: { 'turbo-method': :delete, turbo: true }, id: dom_id(ActiveStorage::Attachment.new, :delete)) {}
```

Comment:
```suggestion
    form_tag('/attachments/:id', method: :delete, data: { turbo: true }, id: dom_id(ActiveStorage::Attachment.new, :delete)) {}
```

Je pense pas que y'a besoin de `turbo-method`